templates/compute/resources/vkcs_compute_instance.md.tmpl

```diff
@@ -23,6 +23,9 @@ Use `vkcs_compute_floatingip_associate` to make the instance accessible from Int
 ~> **Attention:** When an instance connects to a network using an existing port, the instance's security groups are not automatically applied to the port. Instead, the security groups that were initially defined when the port was created will be applied to the port. As a result, the instance's security groups will not take effect in this scenario.
 {{tffile "examples/compute/instance/multiple_networks/main.tf"}}
 
+### Instance with default security group and custom security groups
```

Comment:
default and custom security group - would be worse?

Reply:
fix